docs/backup.md

```diff
@@ -0,0 +1,112 @@
+# Backup
+
+Follow this guide to create backup for elemental configuration installed together with Rancher.
+
+## Install rancher-backup operator for Rancher
+
+Go to official [Rancher documentation](https://docs.ranchermanager.rancher.io/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/back-up-rancher) and install rancher-bakup operator from there.
+
+!!! warning
+    For Rancher v2.7 and below it is needed to edit ResourceSet for rancher-backup operator.
+    For Rancher v2.7.1+ backup will be done automatically by rancher-backup operator and no further operation are needed.
+
+
+## Backup elemental with rancher-backup operator (only for Rancher v2.7 and below)
+
+Fetch `rancher-resource-set` object from Kubernetes cluster
+
+```shell
+kubectl get ResourceSet rancher-resource-set -o yaml > rancher-resource-set.yaml
+```
+
+At the end of this file add the definition of elemental resources
+
```

Comment:
I wonder if we should have a method for this using yq to concatenate the 2 yaml files.

i.e. "save this file as elemental-resource-set.yaml, then use yq to merge the files" instead of manually adding it. Mainly becuase Im wondering if users wont put the resources under the wrong key....yaml is a pita :)

Reply:
I have how to use yq to auto-merge both files